userapi/src/main/java/fk/prof/userapi/exception/ProfException.java

```diff
@@ -0,0 +1,5 @@
+package fk.prof.userapi.exception;
```

Comment:
Also, I think it is the right time to move significant common code to a common module. The problem is that the other ongoing PR and future PR will have to merge these changes (once this one is merged to master). Then this kinds of pollutes those PR too. It ultimately is pushing the refactoring to the end.

We can keep moving common utility code to common module as needed. Right now we can just move the http client there.

For this PR, there was no need for so many changes in userapi.